packages/compass-indexes/src/modules/rolling-indexes-service.ts

```diff
@@ -0,0 +1,48 @@
+import type { AtlasService } from '@mongodb-js/atlas-service/provider';
+import type { ConnectionInfoAccess } from '@mongodb-js/compass-connections/provider';
+import type { CreateIndexesOptions } from 'mongodb';
+import toNS from 'mongodb-ns';
+
+export class RollingIndexesService {
+  constructor(
+    private atlasService: AtlasService,
+    private connectionInfo: ConnectionInfoAccess
+  ) {}
+  async listRollingIndexes(namespace: string) {
```

Comment:
Was going back and forth on whether or not I should also process the result here and map it to something closer to what listing indexes from a db looks like, still unsure whether or not we should, happy to change if there are some opinions about that

Reply:
i think we are doing same in ai-service, just fetching data and processing it elsewhere.